app/reducers/cpacpy.js

```diff
@@ -14,12 +14,15 @@ import {
   CPACPY_SCHEDULER_ONLINE,
   CPACPY_SCHEDULER_OFFLINE,
   CPACPY_SCHEDULER_CONNECT_SEND_CALLBACK,
-} from '../actions/cpacpy'
+  CPACPY_SCHEDULER_ADDNEW,
 
+  CPACPY_CONFIG_LOAD_SUCCESS,
+} from '../actions/cpacpy'
 const initialState = fromJS({
   schedulers: [
     { id: 'local', name: 'Local', version: 'unknown', backends: [], address: scheduler.local, polling: false, detecting: false, online: null, connected: false, connect: { callbacks: {} } },
   ],
+  // id for scheduler
   scheduler: 'local',
```

Comment:
I set it to `latestScheduler`, as we've discussed last Thu. I found that if we want to change the name of the state, we only needed to modify 3 names in `reducers/cpacpy.js`, and there shouldn't be any more place to make modifications.

Reply:
Actually I don't think we need this var (`initialState`) anymore, since we're loading this info from local storage. Even if local storage does not exist, we have a default value in `sagas/cpacpy.default.js`. What do you think?

Reply:
It can be kept, according to the cpacpy selector comments.